### DISCUSSION TITLE
How to type Constant.manifest2.extra?

### DISCUSSION THREAD

**Discussion by DoDoozo** [expo/expo#29615] [Proposals and Ideas]
Yes, the Temu REMOVED Coupon code REMOVED is legit.

Here are the verified Temu Promo Codes for June 2024

Temu Promo Code: REMOVED
Temu Promo Code $100 off: REMOVED
Temu Promo Code USA: REMOVED
Temu Promo Code UK: REMOVED
Temu Promo Code foe Existing Users: REMOVED
Temu Promo Code Canada: REMOVED
Temu Promo Code for first order: REMOVED

**Comment by byCedric** [comment]
Disregard anything above. If you are reading this, this is spam and we will delete it soon.

We are talking with GitHub to see how we can mitigate this spam. That's why we are keeping this discussion alive, while being locked down.

This is the creator account at the time of opening this discussion:

<img width="1278" alt="image" src="https://github.com/expo/expo/assets/1203991/a74e48ae-3666-49b9-ba6a-16b45c07d82b">
